spec/services/zendesk.rb

```diff
@@ -12,10 +12,8 @@
     def mock_zendesk_response(body)
       stub_request(
         :get,
-        /.*zendesk.com\/api\/v2\/.*/
-      ).with(
-        :body => "{\"query\":\"query:created>7days group:\\\"Incident response and tuning\\\" type:ticket\"}"
-      ).to_return(:status => 200, :body => body, :headers => {"Content-Type" => "application/json"})
+        /.*zendesk.com\/api\/.*/
```

Comment:
@andyhd Don't think this test will be executed if you just run `rake` or `rspec`. 

RSpec by default will run files suffixed with `_spec.rb`.

Secondly, we set the `zendesk_verison_url` to include `../api/v2/`, why do we exclude it in the stub?